code/modules/shuttle/shuttle.dm

```diff
@@ -867,6 +884,9 @@ GLOBAL_LIST_INIT(shuttle_turf_blacklist, typecacheof(list(
 	if(distant_source)
 		for(var/mob/M as() in SSmobs.clients_by_zlevel[z])
 			var/dist_far = get_dist(M, distant_source)
+			//Cannot hear shuttles from other shuttles
+			if(M.get_virtual_z_level() != get_virtual_z_level())
```

Comment:
Will players on the station be able to hear a shuttle departing/arriving?

Reply:
The departure sound and arrival sound is played while on the station (departure is played before leaving, during warmup and the arrival sound is played after it docks, so yea)